package.json

```diff
@@ -255,37 +255,49 @@
       "default": "automatic",
       "order": 17
     },
+    "openerSplit": {
```

Comment:
@thomasjo Do you have preference regarding naming? 

- `openerSplit`
- `openerSplitDirection`
- something else?